engine/src/resource/product.h

```diff
@@ -32,6 +32,8 @@ class Product
     std::string name;
     // TODO: Can be a fraction for things such as fuel, water, etc. But not for now.
     Resource<int> quantity;
+
+    // TODO: move to int, x100 and add code to deal with cents.
```

Comment:
we probably should just do whole credits and not concern ourselves with fractional credits
(e.g 1 credit = 1 penny = 1 shilling .... )

Reply:
Yeah. I figured that one out too. Still need a todo.